### ISSUE TITLE
Confluence label under the topic are not displayed

### ISSUE THREAD

**Issue by AlexL777** [chris-kent/Easy-Forums#8]
Hello Developers,

I have with Easyforums a small problem and although the labels in the forum are not displayed. I use Confluence 6.0.6 and the Theme Enterprise. Whether it has something to do with the Confluence 6.0.6 update? Or today there was a new update of the Enterprise Themes and I have it installed.

Can you please check this in the test environment. I can show it to you here at http://confluence.liebrecht-projekte.de:8060/display/FOR/Blogparaden+fuer+Maerz-April+2017.

Thanks in advance.

**Comment by chris-kent** [comment]
Hi there,

By design I do not show the Labels attached to Topic pages.

If this is something that you would like, I can make this optional and let
you configure this in the admin pages? Please let me know and I can
schedule this improvement

Cheers

**Comment by AlexL777** [comment]
Hi @chris-kent ,
Yes, I want this improvement and it would be very exciting if you can implement it. Thanks in advance.

**Comment by chris-kent** [comment]
Hi there,

I have implemented this feature, please see version 2.10.

If you like my plugin, can you please leave a review, I don't have many :(

Cheers
Chris

**Comment by chris-kent** [comment]
Oh by the way, I really loved the sandbox you setup with my plugin, however it looked like the plugin was not licensed or installed.  I would love to see this all working when the plugin is enabled. If you don't mind, when it is running I would love to see it.

Cheers

**Comment by AlexL777** [comment]
Hi,
Thank you for your quick help and implementation of the new features. The labels now work. Thank you.

Unfortunately it is only noticeable to me now that there is an issue for anonymous guests in the forum. The Macro forum can not be rendered and returns a value of zero. Check it out at http://confluence.liebrecht-projekte.de:8060/display/FOR.

Something is still not quite correct or has something to do with my Confluence installation?


I will buy this plugin in any case and also review. The EasyForums are there with me in Confluence already and now it should only the guests to see. Can you please look at this again.

Thanks in advance.

**Comment by chris-kent** [comment]
Hi Alex,

Yes I (you) found a bug, I have never tested Anonymous access. I have fixed this in version 2.11. Please install and let me know how it goes.

Cheers
Chris

**Comment by AlexL777** [comment]
Hi! Thank you for your quick response and fixing the error. I have installed the update now and the anonymous access to the forum is given.

In addition, as I promised, I have a small plugin review at https://marketplace.atlassian.com/plugins/net.kentcom.easy-forums/server/reviews. This will help other users to decide for this plugin.

If I still have plug-in requests for EasyForums, I will gladly turn to you. The plugin I will soon buy, because it makes a good forum and I like it very much.
